src/Libraries/RevitNodes/Elements/FamilyInstance.cs

```diff
@@ -155,6 +155,9 @@ private void InitFamilyInstance(Autodesk.Revit.DB.FamilySymbol fs, XYZ pos)
 
         private void InternalSetLevel(Autodesk.Revit.DB.Level level)
         {
+            if (InternalFamilyInstance.LevelId.Compare(level.Id) == 0)
```

Comment:
Just curious, what is the `InternalFamilyInstance.LevelId` the first time the family instance is created? Does it have some default value before it is set for the first time or is it `null`?